index.html

```diff
@@ -6,42 +6,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+   
     <title>Jonathan's Portfolio</title>
 </head>
-
-<body>
-
-
     <body>
-        <div class="intro-card">
+       
             <header>
                 <div class="nav-container">
                     <div class="nav-logo">
                         <a href="#">My Logo</a>
                     </div>
                     <div class="desktop-tools">
-                        <ul class="flex">
+                        <nav class="content">
+                            <ul class="navnav-menu">
+                            <li class="navnav-item">
+                                <a href="#" class="link">Portfolio</a>
+                            </li>
+                            <li>
+                                <a href="#" class="link">About</a>
+                            </li>
                             <li>
-                                <a href="#">Portfolio</a>
+                                <a href="#" class="link">Contact</a>                            
+                            </li>
+                            </ul>
+                        </nav>
```

Comment:
- [ ] Based on the requirements of this project, the hamburger button and the menu options follow the design from Figma that's font, colors, images, text, and margins should be EXACTLY the same as in Figma design. Kindly visualize the screenshots below and fix :wrench:  the necessary changes.

| Yours | Figma |
| --- |--- |
| ![Screenshot from 2023-01-06 11-08-58](https://user-images.githubusercontent.com/63087491/210982148-c4f8d66c-c252-4b45-b123-c6445d9935da.png) | ![Screenshot from 2023-01-06 11-10-48](https://user-images.githubusercontent.com/63087491/210982380-735955c5-9035-4a3e-8671-529f9b596a18.png) |